numba/core/byteflow.py

```diff
@@ -1046,6 +1262,14 @@ def op_GEN_START(self, state, inst):
         # no-op in Numba
         pass
 
+    def op_BINARY_OP(self, state, inst):
+        op = dis._nb_ops[inst.arg][1]
+        rhs = state.pop()
+        lhs = state.pop()
+        res = state.make_temp()
```

Comment:
Having spent a load of time debugging Python 3.11 now, I think it'd be a good idea to get the name of the operator into `res` so that e.g. a multiply binop ends up something like `$mul_<offset> = binop(fn=<built-in function mul> ... ` in the Numba IR. Else all these store locations end up being called `$<offset>binop`, which makes it harder to debug against. What do you think?